integration-tests/features/component_analysis_v2.feature

```diff
@@ -0,0 +1,198 @@
+Feature: Component analysis v2 API
+  
+  Scenario Outline: Check the component analysis V2 REST API endpoint for components with recommendation
+    Given System is running
+    Given Three scale preview service is running
```

Comment:
Are we not checking for CVE's?

Reply:
we are looking if the CVE's are present we are not looking for specific CVE id's and cvss

Reply:
cve id and score checks added